actionview/lib/action_view/routing_url_for.rb

```diff
@@ -4,21 +4,13 @@
 
 module ActionView
   module RoutingUrlFor
-    # Returns the URL for the set of +options+ provided. This takes the
-    # same options as +url_for+ in Action Controller (see the
-    # documentation for ActionDispatch::Routing::UrlFor#url_for). Note that by default
+    # Returns the URL for the set of +options+ provided. Note that by default
     # <tt>:only_path</tt> is <tt>true</tt> so you'll get the relative <tt>"/controller/action"</tt>
     # instead of the fully qualified URL like <tt>"http://example.com/controller/action"</tt>.
     #
     # ==== Options
-    # * <tt>:anchor</tt> - Specifies the anchor name to be appended to the path.
-    # * <tt>:only_path</tt> - If true, returns the relative URL (omitting the protocol, host name, and port) (<tt>true</tt> by default unless <tt>:host</tt> is specified).
-    # * <tt>:trailing_slash</tt> - If true, adds a trailing slash, as in <tt>"/archive/2005/"</tt>. Note that this
-    #   is currently not recommended since it breaks caching.
-    # * <tt>:host</tt> - Overrides the default (current) host if provided.
-    # * <tt>:protocol</tt> - Overrides the default (current) protocol if provided.
-    # * <tt>:user</tt> - Inline HTTP authentication (only plucked out if <tt>:password</tt> is also present).
-    # * <tt>:password</tt> - Inline HTTP authentication (only plucked out if <tt>:user</tt> is also present).
+    #
+    # Supports the same options as ActionDispatch::Routing::UrlFor#url_for
```

Comment:
I don't think this is right way, since this is an API documentation, so every options should be mentioned here as well.

Reply:
Thank you for the review!

Do you disagree in this specific case, or would you prefer if the documentation would avoid links in general for cases where the options are identical to other methods?

Reply:
This is in general, I would prefer avoid linking to other source even if that is same.